config/babel.prod.js

```diff
@@ -8,23 +8,34 @@
  */
 
 module.exports = {
+  // Don't try to find .babelrc because we want to force this configuration.
   babelrc: false,
   presets: [
+    // let, const, destructuring, classes, modules
     require.resolve('babel-preset-es2015'),
+    // exponentiation
     require.resolve('babel-preset-es2016'),
+    // JSX, Flow
     require.resolve('babel-preset-react')
   ],
   plugins: [
+    // function x(a, b, c,) { }
     require.resolve('babel-plugin-syntax-trailing-function-commas'),
+    // await fetch()
     require.resolve('babel-plugin-syntax-async-functions'),
+    // class { handleClick = () => { } }
     require.resolve('babel-plugin-transform-class-properties'),
+    // { ...todo, completed: true }
     require.resolve('babel-plugin-transform-object-rest-spread'),
-    require.resolve('babel-plugin-transform-react-constant-elements'),
```

Comment:
Yea, I just moved it a few lines down for consistency with dev config (order shouldn’t matter for this transform).